selenium-system-test/src/test/resources/features/airGapTests/Air-gap tests for testing with NS&I.feature

```diff
@@ -0,0 +1,117 @@
+
+    When they choose to go ahead with creating an account
+    Then they see that the account is created
+
+
+  Scenario: Air-gap 4 - All fields are populated with values of minimum length
+    Given an applicant has the following details:
+      | field          | value      |
+      | first name     | A          |
+      | last name      | 4          |
+      | NINO           | <eligible> |
+      | date of birth  | 12/12/1999 |
+      | email address  | a@a        |
+      | address line 1 | a          |
+      | address line 2 | b          |
+      | address line 3 | c          |
+      | address line 4 | d          |
+      | address line 5 | e          |
+      | postcode       | B          |
+      | country code   | 01         |
+
+    When they choose to go ahead with creating an account
+    Then they see that the account is created
+
+
+  Scenario: Air-gap 7 - Wacky postal address
+    Given an applicant has the following details:
+      | field          | value                    |
+      | first name     | AirGap                   |
+      | last name      | Seven                    |
+      | NINO           | <eligible>               |
+      | date of birth  | 12/12/1999               |
+      | email address  | sarah@smith.com          |
+      | address line 1 | C/O Fish 'n' Chips Ltd.  |
+      | address line 2 | The Tate & Lyle Building |
+      | address line 3 | Carisbrooke Rd.          |
+      | address line 4 | Barton-under-Needwood    |
+      | address line 5 | Derbyshire               |
+      | postcode       | W1J7NT                   |
+      | country code   | 04                       |
+
+    When they choose to go ahead with creating an account
+    Then they see that the account is created
+
+
+  Scenario: Air-gap 11 - Populate forename, surname, address lines (phone number) and email with accented characters
+    Given an applicant has the following details:
+      | field          | value                    |
+      | first name     | AirGapRené Chloë         |
+      | last name      | ElevenO'Connor-Jørgensen |
+      | NINO           | <eligible>               |
+      | date of birth  | 12/12/1999               |
+      | email address  | sarah@smith.com          |
+      | address line 1 | 17 Ålfotbreen            |
+      | address line 2 | Grünerløkka              |
+      | address line 3 | Bodø                     |
+      | address line 4 | Hørdy-Gürdy4             |
+      | address line 5 | Hørdy-Gürdy5             |
+      | postcode       | 19023                    |
+      | country code   | 02                       |
+
+    When they choose to go ahead with creating an account
+    Then they see that the account is created
```

Comment:
i see only happy path scenarios above, cant we test non-happy path cases ?

Reply:
We can't test the unhappy paths yet because we don't have the ability to make mandatory fields blank - this issue is blocking https://jira.tools.tax.service.gov.uk/browse/HTS-461 too.

Reply:
I'm going to merge this in, we can add unhappy cases later